index.js

```diff
@@ -787,13 +786,18 @@ function speedTest(options) {
 module.exports = speedTest;
 
 function visualSpeedTest(options, callback) {
+  // We only need chalk and DraftLog here. Lazy load it.
+  var chalk = require('chalk')
+  var DraftLog = require('draftlog').into(console)
```

Comment:
Variable isn't used anywhere so please remove it. Also, a semi for consistency please.